gips/0065-subgraph-availability-manager.md

```diff
@@ -0,0 +1,123 @@
+
+The number of oracle operators is strictly limited to five, ensuring a balance between decentralization and manageability.
+
+## 3. Responsibility and Integrity
+
+Operators must ensure their oracles remain operational, acknowledging that while occasional outages are part of normal operations, efforts to minimize these are crucial. Operators should communicate their infrastructure choices with one another to prevent risks associated with common dependencies, thereby enhancing the network's overall resilience. Active participation in a group chat to discuss operational challenges and coordinate on mitigating risks is required. Operators who are unresponsive or consistently underperform may be recommended for replacement by the technical advisory board or fellow operators to the Graph Council.
+
+## 4. Subgraph Availability Oracle Software and Versioning
+
+Subgraph Availability Oracles operators must use the software available at [The Graph's Subgraph Oracle repository](https://github.com/graphprotocol/subgraph-oracle). This ensures uniformity and accuracy in their assessments across the network. A specific commit or release version from this repository will be designated as the required version for all oracles. This version is subject to change over time as updates and improvements are made.
+
+Oracle operators are responsible for reaching consensus on the specific commit and configuration to be used, and must align their operations with these agreed-upon standards by posting a transaction with the details to a `DataEdge` smart contract. A dedicated subgraph will track these declarations, flagging any deviations for prompt resolution to maintain network consistency.
+
+The transaction to the `DataEdge` should follow the format:
+
+```json
+{
+    "commit-hash": "XXXXX",
+    "config": {
+        "ipfs-concurrency": "4",
+        "ipfs-timeout": "10000",
+        "min-signal": "100",
+        "period": "300",
+        "grace_period": "0",
+        "supported-networks": "mainnet,gnosis,xdai,eip155:1,eip155:100,avalanche,eip155:43114,arbitrum-one,eip155:42161,celo,eip155:42220,fantom,eip155:250,matic,eip155:137",
+        "supported_data_source_kinds": "ethereum,ethereum/contract,file/ipfs,substreams,file/arweave",
+        "subgraph": "https://api.thegraph.com/subgraphs/name/graphprotocol/graph-network-arbitrum",
+        "subgraph_availability_manager_contract": "CONTRACT_ADDRESS",
+        "oracle_index": "ORACLE_INDEX",
+    }
+}
+
+```
+
+It is strongly recommended that Oracle operators diversify their sources for `IPFS_NODE_URL` and `RPC_URL` configurations. Utilizing varied sources for these critical components not only enhances the robustness and reliability of the network but also mitigates potential risks associated with single points of failure.
```

Comment:
Could you explain `IPFS_NODE_URL` and `RPC_URL`? Specifically, I'm curious about how varying these helps ensure that the "robustness and reliability of the network."